da_test.go

```diff
@@ -90,3 +94,36 @@ func CheckErrors(t *testing.T, da da.DA) {
 	assert.Error(t, err)
 	assert.Empty(t, blob)
 }
+
+func GetIDsTest(t *testing.T, da da.DA) {
+	msgs := [][]byte{[]byte("msg1"), []byte("msg2"), []byte("msg3")}
+
+	ids, proofs, err := da.Submit(msgs)
+	assert.NoError(t, err)
+	assert.Len(t, ids, len(msgs))
+	assert.Len(t, proofs, len(msgs))
+
+	found := false
+	end := time.Now().Add(1 * time.Second)
+	for i := uint64(1); !found && !time.Now().After(end); i++ {
+		ret, err := da.GetIDs(i)
+		if err != nil {
+			break
```

Comment:
nit:
I think there is more to learn by making this `t.Error("unable to get ids:",err)`
The test would still fail, and we might see something in certain heights failing.